docs/lib/datastore/getting-started.md

```diff
@@ -1,52 +1,136 @@
 
 ### Sample schema
 
-For the purpose of this guide, let's use the following simple GraphQL schema:
+Your first step is to define a schema for a small blog application. It has a single model, a `Post`. New types and constructs will be added to this base schema as more concepts are presented.
 
-```graphql
-enum PostStatus {
-  ACTIVE
-  INACTIVE
-}
+Open the `schema.graphql` file located by default at `amplify/backend/{api_name}/`  and **define the model** `Post` as follows.
 
+```graphql
 type Post @model {
   id: ID!
   title: String!
-  rating: Int!
-  status: PostStatus!
+  content: String
 }
 ```
 
-<inline-fragment platform="js" src="~/lib/datastore/fragments/js/getting-started/generate-models.md"></inline-fragment>
-<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/generate-models.md"></inline-fragment>
-<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/generate-models.md"></inline-fragment>
+## Install Amplify Libraries
+
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/20_installLib.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/20_installLib.md"></inline-fragment>
+
+## Idiomatic persistence: models
+
+Amplify DataStore relies on platform standard data structures to represent the data schema in an idiomatic way. The persistence language is composed by data types that satisfies the `Model` interface and operations defined by common verbs such as `save`, `query` and `delete`.
+
+In order to start choose an option to **generate models** from the `schema.graphql`.
+
+### Option 1: Use IDE integration
+
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/30_amplifyTools.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/30_amplifyTools.md"></inline-fragment>
+
+### Option 2: Use Amplify CLI
+
+If more control over the configuration and code generation process is needed the Amplify CLI can be used directly. This option is also useful for **existing codebases** with Amplify already configured.
+
+The base structure for a DataStore app is created by adding a new GraphQL API to your app.
+
+```console
+amplify add api
+```
+
+During the API configuration process select **GraphQL** as the API type and reply to the questions as follows. Make sure you respond **yes** to **advanced settings** and turn on **conflict detection**. This setting is critical when syncing data to the cloud since the conflict resolution strategy is what allows local data to be reconciled with data from the cloud backend.
+
+```console
+? Please select from one of the below mentioned services:
+    `GraphQL`
+? Choose the default authorization type for the API
```

Comment:
Seemsl ike we forgot a line, i think this should be:
```
? Please select from one of the below mentioned services:
    `GraphQL`
? Provide API name:
    `mydatastoreapp`
? Choose the default authorization type for the API
    `API key`


... then things.. go off the rails here:


? Select the default resolution strategy
   Auto Merge
? Do you have an annotated GraphQL schema?
   No
? Do you want a guided schema creation?
   No
? Provide a custom type name (MyType)
   ???
```

Reply:
fixed